VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -45,6 +45,7 @@
     public static final String EXTRA_IS_RESPONSE = "is_response";
     public static final String EXTRA_DISCUSSION_COMMENT = "discussion_comment";
     public static final String EXTRA_DISCUSSION_TOPIC_OBJ = "discussion_topic_obj";
+    public static final String EXTRA_DISCUSSION_TOPIC_IS_CLOSED = "discussion_topic_is_closed";
```

Comment:
This should be `EXTRA_DISCUSSION_THREAD_CLOSED`. Also the previous item should be changed from `EXTRA_DISCUSSION_TOPIC_OBJ` to `EXTRA_DISCUSSION_THREAD_OBJ`.